src/index.js

```diff
@@ -149,12 +149,12 @@ async function readFiles(sources) {
                 const data = await parseFile(absFilePath);
                 result.push(_.assign({
                     __metadata: {
-                        id: `${relProjectPath}`,
+                        id: `${convertPathToPosix(relProjectPath)}`,
                         source: SOURCE,
                         sourceName: name,
-                        sourcePath: sourcePath,
-                        relSourcePath: relSourcePath,
-                        relProjectPath: relProjectPath
+                        sourcePath: convertPathToPosix(sourcePath),
+                        relSourcePath: convertPathToPosix(relSourcePath),
+                        relProjectPath: convertPathToPosix(relProjectPath)
                     }
                 }, data));
```

Comment:
@ramonsnir could there be inconsistent paths in windows when the provided `sourcePath` is posix like (e.g.: `content/pages`) but when the code above is run `path.resolve` returns mixed paths and then `join` also adds system specific separators:
```
// sourcePath = 'content/pages'
const absSourcePath = path.resolve(sourcePath);
// is absSourcePath now === 'C:\\project\\contnet/pages'
// or 'C:\\project\\contnet\\pages'
const absFilePath = path.join(absSourcePath, filePath);
// is the absFilePath now has mixed separators to 'C:\\project\\contnet/pages\\index.md'
```

Also, can the `convertPathToPosix` be replaced with [path.normalize(path)](https://nodejs.org/api/path.html#pathnormalizepath)

Reply:
This is not what `path.normalize(path)` does at all, you're free to test that.

Reply:
I don't see the issue that you're referring to with the mixed separators.